impl/client/src/main/javascript/web/prompting/components/ScopedPentahoButtonComponent.js

```diff
@@ -75,9 +77,9 @@ define([ 'cdf/components/BaseComponent', 'common-ui/jquery-clean' ], function(Ba
      *
      * @name ScopedPentahoButtonComponent#expression
      * @method
-     * @param {Boolean} isInit
+     * @param {Boolean} isInit Flag
```

Comment:
Hard to understand, can you add a little bit more description for the isInit parameter?